geolocator_android/android/src/main/java/com/baseflow/geolocator/location/LocationManagerClient.java

```diff
@@ -161,27 +153,40 @@ public void startPositionUpdates(
     this.positionChangedCallback = positionChangedCallback;
     this.errorCallback = errorCallback;
 
-    LocationAccuracy locationAccuracy =
-        this.locationOptions != null ? this.locationOptions.getAccuracy() : LocationAccuracy.best;
-
-    this.currentLocationProvider = getBestProvider(this.locationManager, locationAccuracy);
-
-    if (this.currentLocationProvider.trim().isEmpty()) {
-      errorCallback.onError(ErrorCodes.locationServicesDisabled);
-      return;
-    }
-
+    LocationAccuracy accuracy = LocationAccuracy.best;
     long timeInterval = 0;
     float distanceFilter = 0;
+    @LocationRequestCompat.Quality int quality = LocationRequestCompat.QUALITY_BALANCED_POWER_ACCURACY;
     if (this.locationOptions != null) {
       timeInterval = locationOptions.getTimeInterval();
       distanceFilter = locationOptions.getDistanceFilter();
+      accuracy = locationOptions.getAccuracy();
+      quality = accuracyToQuality(accuracy);
```

Comment:
Rereading the [documentation](https://developer.android.com/reference/androidx/core/location/LocationRequestCompat#PASSIVE_INTERVAL()), we should use the `PASSIVE_INTERVAL` here when the `LocationAccuracy` is set to lowest:

```suggestion
      accuracy = locationOptions.getAccuracy();
      distanceFilter = locationOptions.getDistanceFilter();
      quality = accuracyToQuality(accuracy);
      timeInterval = (accuracy == LocationAccurracy.lowest) 
          ? LocationRequestCompat.PASSIVE_INTERVAL 
          : locationOptions.getTimeInterval();
```